app/views/admin/unattached_promo_registrations/index.html.slim

```diff
@@ -103,20 +103,20 @@
               . = date_select :referral_code_report_period, :start
             .flex-two.align-self-flex-center
               . = label_tag "Report period end"
-              . = date_select :referral_code_report_period, :end
+              . = date_select :referral_code_report_period, :end, default: 1.day.from_now
```

Comment:
So I think what `1.day.from_now` does is take the current day in PST and add one day to it.  What I think we want is to have it be the current day, but in UTC time.

Reply:
This is intentional. Preferably we do 2 days from now (I don't in what case it would happen) but I think this "good-enough" solution is ideal. 

Alternatively, if we start supporting promo codes in Japan/South Korea/Australia/China, the UTC date is wrong.

Reply:
Why is showing the date 2 days from the current time PST more preferable than the exact day that the referral server will be recording the downloads at the current time?  Seems like this would be adding ambiguity.

Don't we already support those countries? Why would the UTC date be wrong?

Reply:
I'm just highlighting that we should show more info rather than less.

Supposing that the promo server saves everything in UTC, then yes, you're right.